### PR TITLE
More camera changes

### PR DESCRIPTION
**Pull request by Anup0101**

### CODE REVIEW

**Comment by Anup0101** [comment]
Since this hasn’t been tested yet, I’d like to point out that you just need to choose a primary vs secondary camera instance and assign the width/height you want for the captured image. All else for the camera2 implementation should be good here.

**Comment by Davidezrajay** [comment]
@Anup0101 thanks for this. Can you test it and make sure it works?

**Comment by Anup0101** [comment]
No prob, I can look into it this weekend.

File: app/build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ android {
     }
     defaultConfig {
         applicationId "org.greenstand.android.TreeTracker"
-        minSdkVersion 15
+        minSdkVersion 21
```
Comment (Elforama) [inline app/build.gradle]: This app is for 3rd world countries. Do we really want to ignore users with old phones?

**Comment by Elforama** [comment]
I'm going to close this PR. Updates to the camera are now being worked on using the CameraX jetpack library.

**Pull request closed without merging**